packages/gestalt/src/GestaltProvider.js

```diff
@@ -10,7 +10,7 @@ import {
 type Props = {|
   children: React.Node,
   colorScheme?: ColorScheme,
-  id: ?string,
+  id?: string,
```

Comment:
@jennyscript fyi

Discovered this issue when trying to type `<GestaltProvider />` without an id.